public/app/features/datasources/components/EditDataSourceActions.tsx

```diff
@@ -1,71 +1,58 @@
-import { css } from '@emotion/css';
 import React from 'react';
 
-import { GrafanaTheme2 } from '@grafana/data';
 import { config } from '@grafana/runtime';
-import { LinkButton, useStyles2 } from '@grafana/ui';
+import { Button } from '@grafana/ui';
 import { contextSrv } from 'app/core/core';
 import { AccessControlAction } from 'app/types';
 
 import { useDataSource } from '../state';
 import { trackCreateDashboardClicked, trackExploreClicked } from '../tracking';
 import { constructDataSourceExploreUrl } from '../utils';
 
-const getStyles = (theme: GrafanaTheme2) => {
-  return {
-    button: css({
-      marginLeft: theme.spacing(2),
-    }),
-  };
-};
-
 interface Props {
   uid: string;
 }
 
 export function EditDataSourceActions({ uid }: Props) {
-  const styles = useStyles2(getStyles);
   const dataSource = useDataSource(uid);
   const hasExploreRights = contextSrv.hasPermission(AccessControlAction.DataSourcesExplore);
 
   return (
     <>
-      <LinkButton
-        icon="apps"
-        fill="outline"
-        variant="secondary"
-        href={`dashboard/new-with-ds/${dataSource.uid}`}
-        onClick={() => {
-          trackCreateDashboardClicked({
-            grafana_version: config.buildInfo.version,
-            datasource_uid: dataSource.uid,
-            plugin_name: dataSource.typeName,
-            path: location.pathname,
-          });
-        }}
-      >
-        Build a dashboard
-      </LinkButton>
-
       {hasExploreRights && (
-        <LinkButton
-          icon="compass"
-          fill="outline"
+        <Button
+          type="button"
+          size="sm"
           variant="secondary"
-          className={styles.button}
-          href={constructDataSourceExploreUrl(dataSource)}
           onClick={() => {
             trackExploreClicked({
               grafana_version: config.buildInfo.version,
               datasource_uid: dataSource.uid,
               plugin_name: dataSource.typeName,
               path: location.pathname,
             });
+            location.href = constructDataSourceExploreUrl(dataSource);
           }}
         >
-          Explore
-        </LinkButton>
+          Explore data
+        </Button>
       )}
+      <Button
+        type="button"
+        size="sm"
+        variant="secondary"
+        onClick={() => {
+          trackCreateDashboardClicked({
+            grafana_version: config.buildInfo.version,
+            datasource_uid: dataSource.uid,
+            plugin_name: dataSource.typeName,
+            path: location.pathname,
+          });
+          location.href = `dashboard/new-with-ds/${dataSource.uid}`;
+        }}
+      >
+        Build a dashboard
+      </Button>
```

Comment:
I am not sure about this change - did we confirm that we are losing tracking of events because of navigating away? I think Rudderstack implemented using `navigator.sendBeacon()`, so those requests won't get cancelled once the page is navigating away.)

My concern with this change is that it stops behaving like a regular link, people cannot Ctrl+Click / Cmd+Click it anymore, or Right-click -> Open in new tab.

Reply:
LinkButton should support onClick so not sure why we need to change it to Button